test/IntentTest.py

```diff
@@ -70,6 +71,24 @@ def test_at_least_one(self):
             assert result_intent.get('PlayVerb') == 'play'
             assert result_intent.get('Radio Station') == "barenaked ladies"
 
+    def test_at_least_one_with_tag_in_multiple_slots(self):
```

Comment:
Ok, so to be clear, in the past this failed, and the behavior we want to assert on is that if the tag appears in multiple slots, the parse result with the largest phrase coverage wins. Sound right?

Reply:
Yes, that is correct.